spec/controllers/criteria_controller_spec.rb

```diff
@@ -250,7 +248,7 @@
             post_as admin,
                     :create,
                     params: { assignment_id: assignment.id, flexible_criterion: { name: 'first', max_mark: 10 },
-                              new_criterion_prompt: 'first', criterion_type: 'FlexibleCriterion' },
+                              new_criterion_prompt: 'first', criterion_type: flexible_criterion.class.to_s },
```

Comment:
@ashih2018 the `create` method is the one place where we still need to keep `criterion_type` as a param. This is the current way we allow the user to choose what criterion type they actually want to create! But then once the criterion is created and we have an id, we don't need to pass the criterion type for the other methods (and you've correctly removed them).

Reply:
@david-yz-liu I realized this once I removed it, I put up a PR to keep track of the changes I've currently made, is there a proper way to put one up as a draft PR for my own use until it is ready to be looked at?